### ISSUE TITLE
Insight context menu is not keyboard accessible

### ISSUE THREAD

**Issue by felixfbecker** [sourcegraph/sourcegraph#21517]
I can open the menu by hitting <kbd>space</kbd>, but arrow keys nor <kbd>tab</kbd> do not select different items, or at least there is no visual indication. I tried pressing <kbd>enter</kbd> (I thought that may "enter" the menu) and it immediately deleted the insight, which is really dangerous. (related: #21515).

The behavior should follow the spec for the ARIA Menu role with regards to keyboard navigation and there should be visual indication of the selected item.

**Comment by vovakulikov** [comment]
Close this since that problem has been fixed since we use a new accessible context menu component
